src/common/components/cookieConfigurationProvider/useCookieConfigurationContext.tsx

```diff
@@ -0,0 +1,7 @@
+import { useContext } from 'react';
+
+import cookieConfigurationContext from './CookieConfigurationContext';
+
+export default function useCookieConfigurationContext() {
```

Comment:
I would remove the "context" appendix

Reply:
Also, instead of "configuration" we could use just "config".

Reply:
Renamed, rebased & [force pushed](https://github.com/City-of-Helsinki/kukkuu-ui/compare/19dce86b3d8e5a71906c0d8b67857f0af263f066..f9660b36648ea58d38e4712887a3e46eba71235e).